runtime/altair/src/lib.rs

```diff
@@ -1676,6 +1671,10 @@ impl fp_self_contained::SelfContainedCall for RuntimeCall {
 
 	#[cfg(not(feature = "testnet-runtime"))]
 	fn check_self_contained(&self) -> Option<Result<Self::SignedInfo, TransactionValidityError>> {
+		use pallet_ethereum::TransactionAction;
+		use runtime_common::evm::GetTransactionAction;
+		use sp_runtime::transaction_validity::InvalidTransaction;
+
```

Comment:
It seems that `cargo` doesn't handle `#[cfg(not` rules sensibly when it comes to imports, because the warnings we get are not actually an issue 🤷‍♂️  but by moving the imports here everyone is happy